### ISSUE TITLE
opening links , audio, and DM UI

### ISSUE THREAD

**Issue by rkingett** [manuelcortez/TWBlue#60]
TW Blue does not open links in the invisable interface when I press the key commands to do so, on any kind of link. Also, the DM UI window does not show anything when I send a direct message. I have to navigate it with NVDA because it looks like a black window that pops up. Also, I am unable to attach a recorded audio file. after recording audio, the attach button is greyed out

**Comment by manuelcortez** [comment]
When you was recording  audio did you pressed the stop button?

**Comment by rkingett** [comment]
the stop button was greyed out

On 8/21/2015 11:06 AM, Manuel Cortéz wrote:

> When you was recording audio did you pressed the stop button?
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/manuelcortez/TWBlue/issues/60#issuecomment-133475584.

## 

Robert Kingett, Journalist

View my resume https://blindjournalist.wordpress.com/resume/

view my blog https://blindjournalist.wordpress.com/

View my Gaming articles http://gamingeyes.tumblr.com/
